jenkins/bootstrap.py

```diff
@@ -478,6 +478,15 @@ def find_version(call):
         ]
         return call(cmd, output=True).strip()
 
+    # Make up some tag instead of unknown
+    cmd = ['git', 'describe', '--always', '--tags', '--abbrev=14']
+    try:
+        tag = call(cmd, output=True)
+        if tag:
+            return '+%s' % tag.strip()
```

Comment:
oh right, you mentioned testgrid. does it do the right thing here?

alternately: we already save `repo` in the metadata section of `finished.json` - maybe we should have a new field `git-commit` too?

which is what @rmmh just said